Tweak diff colours

danfo opened the pull request:
I spent some time cleaning up how a diff displays in Nord. Let me know what you think.

**Before:**
![scrot-feature-ui-diffeditor-removed-2](https://user-images.githubusercontent.com/99922/48679422-b0e2a580-eb98-11e8-8c24-bb06edd2f3bf.png)
![scrot-feature-ui-diffeditor-inserted-2](https://user-images.githubusercontent.com/99922/48679421-ae804b80-eb98-11e8-9c83-08be695b63d8.png)

**After:**
![scrot-feature-ui-diffeditor-removed](https://user-images.githubusercontent.com/99922/48679426-b7711d00-eb98-11e8-9b01-8daacafb0530.png)
![scrot-feature-ui-diffeditor-inserted](https://user-images.githubusercontent.com/99922/48679429-bb9d3a80-eb98-11e8-8af9-5f97d2626a04.png)

arcticicestudio commented on themes/nord.json:
```diff
@@ -238,7 +236,7 @@
       "scope": "comment",
       "settings": {
-        "foreground": "#4C566A"
+        "foreground": "#aaddff40"
```
I think this is a personal change so please it, there is also already a ongoing change to improve the comment color contrast/brightness at arcticicestudio/nord#94.

arcticicestudio commented on themes/nord.json:
```diff
@@ -14,10 +14,8 @@
     "button.hoverBackground": "#4c566a",
-    "diffEditor.insertedTextBackground": "#a3be8c4d",
-    "diffEditor.insertedTextBorder": "#a3be8c4d",
+    "diffEditor.insertedTextBackground": "#a3eeaa2d",
```
Let's get this done and change it to `#81a1c133` so we can leave these `nord14` opacity-variations behind.

arcticicestudio commented on themes/nord.json:
```diff
@@ -14,10 +14,8 @@
     "button.foreground": "#d8dee9",
     "button.hoverBackground": "#4c566a",
-    "diffEditor.insertedTextBackground": "#a3be8c4d",
-    "diffEditor.insertedTextBorder": "#a3be8c4d",
```
:+1: for removing these borders, looks by far cleaner and more clearer.

arcticicestudio commented on themes/nord.json:
```diff
@@ -417,7 +415,7 @@
       ],
       "settings": {
-        "foreground": "#4C566A"
+        "foreground": "#aaddff40"
```
Same for this personal change, please remove it.

danfo commented:
You're welcome @arcticicestudio. Thanks for Nord, it's a joy to use every day! 

Nice site. It does help for understanding the big picture, very clear now how it doesn't make sense to accept colour palette tweaks within one theme 🙂 

### Added lines colour
I would like to help get this right. I think added lines on a diff are clearest to see when there is some hint of green. There's something primitive and natural going on here,

```diff
- removed 🔥
+ added 🌱 
```

And it can still be subtle and fresh... How about some opacity of `nord7`?

### Comment colour
I _really_ like the comment colour and subtlety, and didn't intend to change the visible colour. The problem is, diffs are sections of a lighter background colour. Let me illustrate,

<img width="960" alt="Screen Shot 2019-03-24 at 9 26 46 pm" src="https://user-images.githubusercontent.com/99922/54884634-0a771b80-4e7c-11e9-9397-c8303377afd8.png">

<img width="957" alt="Screen Shot 2019-03-24 at 9 29 59 pm" src="https://user-images.githubusercontent.com/99922/54884636-0e0aa280-4e7c-11e9-9bc8-67fa2ed7fa0d.png">

We can solve this by not flattening the opacity into an RGB hex code; I think comments look best when that subtle opacity is preserved in RGBA.

Maybe we can match the existing flattened comment hex code with a low opacity Snow Storm?

arcticicestudio commented:
@danfo Can you please test the theme using the `develop` branch again and check if this is still the case for inline changes? I've merged #118 that increases the comment color by default so maybe this will also improve the readability when using `nord9` as diff color.

Also @octref has also [reported this _diff_ problem][oct_com] and [created a improvement issue for VS Code][i] based on [my thinking aloud comment about how this could be solved][c] :smile: 
If using `nord9` still doesn't solve completely the problem I think it might be a good solution to merge the changes anyway and adapt to the suggested solution when the improvement might be implemented in VS Code.

[oct_com]: https://github.com/arcticicestudio/nord-visual-studio-code/issues/14#issuecomment-479306223
[i]: Microsoft/vscode/issues/71663
[c]: https://github.com/arcticicestudio/nord-visual-studio-code/issues/14#issuecomment-479325318

danfo commented:
Yep readability is OK with `nord9`. Hopefully soon the situation will be that a green such as `nord7` will work as a diff background colour.
